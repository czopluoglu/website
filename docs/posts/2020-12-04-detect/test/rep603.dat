01010001101100101111
01101110011111100111
10100101111110111111
00101000001101000010
10000001110010010100
11010001100110111111
01010011111100111111
00011000011100100010
11111111111101111111
01011000011010100000
01000000010001000101
00111001111111101110
00111011111000101111
11111110111110111011
00111110001111111110
00000100000111011111
11001110011010111010
10010110011011110111
01101101000111110111
11010010110010101101
11110111010110101011
10010011011100000010
10101101111111100010
00001110111011111111
01111000110001100010
01111101111011111011
10000111111011110110
01000101100000111010
00010001111110111101
00000100100111111010
01111111110111111111
11111111111100001101
11011011110110001010
01110111111110001011
10010011011010000010
01110010011001101010
01000011010110101010
11000001101110000111
11011000001111100110
11101001010111110101
10100010100000100110
10011010111111101110
10000100111000101111
11000000101000000011
10110001101111000100
11000101001111110011
00111111111100101100
00000000000110100010
10010100111001000111
10101101111110000010
00110000011000010010
00110101111111101111
11000001011000101101
10111100101110101111
11110001000100101011
11000011011111111111
00101000000010110100
01100101100110010110
00001000010000101101
00000100011110111111
01100001010100110011
00110101111011110110
01100001101101111111
00000010011010001001
11101011011111011000
01110111111000110110
11110111110110100100
00001001101100101001
10000010111110001011
00000100011111100111
01001100110000000001
10100110101000100111
00001110100001001110
10111001001111000010
10000011111101011110
00000000100010000011
01001000101110001001
01001000101100000000
11111010111011101111
10111111111100001010
11010011000010000110
11100110111111110111
00101100101111011110
00110101111111101110
10011111011111111110
11100111111010111111
10101111101111011010
11101110000101001001
01010100010100100011
00100001110010101110
00000010011010111000
01011100100110000010
11101111111111111111
00001010000010101111
01100111010011000111
11100010000100101010
01100011100110001110
11110101011110011010
00011011111111001110
00101100100000110111
01000001001010010011
11000001111011111011
10010111011100101111
10001101101001110100
11001011110011000000
11001011111110110011
10110100110100000110
10000100011100100111
00000101011110111011
01101001111101110010
11011111101111111110
00101101110100110111
11101001011110101010
01101001001010000010
00000100111000010110
01111101111010100110
10011100010111100110
10000001001011000010
00000100101100110111
00101111000011101110
01000001001100111010
10110111111111101111
11111101101111101111
00011011000100001010
10001101111111011110
10101101111011110110
11101111011110100111
00000100011001101111
00111001111000011111
00111101110010100111
11101111000100100110
11000101111000010000
01111111111111000110
01100001101000000100
10100001011001111101
00101000011100111011
11010001011010110101
10011000111010000010
11011001110111111101
11010001011011000111
01000000011011100011
10101000110001101101
00111111111100010010
10110100000110100010
01000001101000011011
01111000100000110010
10111000011011001110
01011000000110000010
01011101101111101111
11100000010100100111
11100110110111100011
10000101001010101110
01000010001010111010
11001011101110111001
11000011101111001110
10001001101101101110
11110001011000101011
11110110101000100111
10010001111000110101
11010001001110101111
11101010011110101000
10001101011111100111
01100001011110100111
10111111111011011111
11000001101110111110
11111110011111101110
10100001011010101001
00010001011000101110
01100111110011011111
10111111110000110100
10101100111111011111
11010000001010100110
11011101111110100110
01101000111110010000
00101101101011001110
11111111111110110011
00001101010010101011
00110000011101111111
11110010100101010110
00001001101100100011
00100000001100101010
11101101111101101111
10100011111111111111
10110100100000000000
11011110111100101101
11010101001000000101
00000101111111011111
10100111010010011010
11000111111011111111
01010111110001000010
00011110010011111010
10110110111110101000
00100100111110110110
10001001010010110010
11010111011010001111
11101101111110101110
10001010111000101000
01111111110111110111
01000001000100101011
00111011110001101111
01000010111001000111
10101101011011111110
11010000010010011010
10100101111110011011
01000000111000010001
00000011101010110111
10101100011100110100
10111110111101111011
00111011010011000011
00101000100000111000
11000011001100110110
10011010111010110001
10000001110110000011
00001000101000101111
10010110011000110101
11111111111000010100
00000001010011001010
10111001111111110011
10011110111101101010
00010001001111001110
11101001111001101111
01000010111010000110
00001010111010100110
11010001111110100010
11000010000010110010
00100011011011100111
00001001101010110110
01110111111000110011
10000000011001010000
01011101010011100111
10110100011111111100
00000001001000101011
01111111011010101100
10000101001111111111
00001000000110000010
01001000000110111111
10011011011111111111
10000111011011100111
01000100001110011010
10011000101101110111
10000001011111001110
10001000100110100110
01010001001010000110
00011001001000000110
11010000111011010001
10100001001000000110
11101110101010101010
10110100100101011111
01110001001110111100
11101001111110101111
