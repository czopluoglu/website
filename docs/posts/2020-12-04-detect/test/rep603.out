-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep603.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.62
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7172
                   IDN index value:   0.6947
                           Ratio r:   0.5625

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    1    3    3    1    2    3    1
    4    2    2    3    4    4    4    3    2    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    4    7   10

   -----------CLUSTER  2-------------

    2    8   12   13   19

   -----------CLUSTER  3-------------

    3    5    6    9   14   18

   -----------CLUSTER  4-------------

   11   15   16   17   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++++++-------+----
   +d+++----++-+-+-----
   ++d++----++++-----+-
   +++d---+-+-+-+-+---+
   +++-d+++-+---------+
   +---+d-+++--+-++--+-
   +---+-d++++----++-+-
   +--++++d+----+---++-
   -----+++d+-+-++-----
   -++++++-+d+++---+---
   -++---+--+d+++--+---
   --++----+++d+++--+--
   -++--+---+++d-++-+--
   ---+---++-++-d+--++-
   -+---+--+--+++d++--+
   +--+-++-----+-+d++++
   ------+--++---++d+++
   -------+---+++-++d-+
   --+--+++-----+-++-d+
   ---++---------+++++d
 
 No cross validation for this DETECT run
